### PR TITLE
Add Summer Dresses

### PR DESCRIPTION
**Pull request by DreamlyJack**
## About the PR

These dresses are an addition to Hawaiian shirts designed for beautiful women
## Why / Balance

Visual aesthetics, as well as summer themes.
## Media

Скриншот 19-06-2024 211639

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl: DreamlyJack
- add: Summer Dresses.

### CODE REVIEW

**Comment by DreamlyJack** [comment]
The idea originated from the Hawaiian shirts to make this kind of unique clothing for women in the Vending clothe machine

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml [hunk 1]
```diff
@@ -46,6 +46,10 @@
     ClothingUniformJumpsuitHawaiBlue: 2
     ClothingUniformJumpsuitHawaiRed: 2
     ClothingUniformJumpsuitHawaiYellow: 2
+    ClothingUniformJumpskirtBlueDress: 2
+    ClothingUniformJumpskirtYellowDress: 2
+    ClothingUniformJumpskirtBlackDress: 2
+    ClothingUniformJumpskirtRedDress: 2
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml:52]: There are 10 comments in this file specifically requesting not to add more so please do not ignore it.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
This needs to use colouring not copy-paste sprites.

**Comment by beck-thompson** [comment]
> This needs to use colouring not copy-paste sprites.

Look how gloves do it if you don't know how (Its not too hard)! See:
https://github.com/space-wizards/space-station-14/blob/b5f6aa0c9d2b3264199a4160ba86f5282c11a316/Resources/Prototypes/Entities/Clothing/Hands/colored.yml

**Comment by DreamlyJack** [comment]
> This needs to use colouring not copy-paste sprites.

I didn't quite understand this sentence. Can you tell us in more detail and suggest a solution to the problem?

**Comment by DreamlyJack** [comment]
do you mean to make them randomly generated?

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml (hunk 1 above)
Comment (DreamlyJack) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml:52]: > There are 10 comments in this file specifically requesting not to add more so please do not ignore it.

Hmm, then how do I solve the problem of getting these dresses in the game? Add them to the theater?

**Comment by lzk228** [comment]
> This needs to use colouring not copy-paste sprites.

they are not the same like colored jumpsuits

**Comment by deltanedas** [comment]
they arent copy paste sprites they have unique shaping

**Comment by DreamlyJack** [comment]
ah, I thought it was obvious that each dress was drawn differently, and was not a copy-paste
I'm more interested in how to solve the problem with the clothes vending machine

**Comment by DreamlyJack** [comment]
I moved the dresses to the dresser

**Comment by MureixloI** [comment]
🙂‍↕️

**Comment by DreamlyJack** [comment]
@metalgearsloth What do you think about this?

**Comment by metalgearsloth** [comment]
Still needs to use sprite colouring.

**Comment by DreamlyJack** [comment]
> Still needs to use sprite colouring.

can I just give each dress 4 variations of its colors, according to the principle of colored jumpskirts?

**Comment by DreamlyJack** [comment]
@metalgearsloth 
I don't think it's a good idea to make dresses colorful.
Скриншот 25-06-2024 103642

**Comment by DreamlyJack** [comment]
or you will have to make dresses of different colors not through the code, because it colors all areas of the sprite

**Comment by beck-thompson** [comment]
You can do two different layers! Have one be just the colored part and the other the white part. I don't quite know if that would look better but I think it would b
image

For example, in my glue and lube PR I had the nozzle be color able but I had the body on a different layer so it could just be white.

image

**Comment by DreamlyJack** [comment]
Good idea, I'll try

**Comment by DreamlyJack** [comment]
!

[Uploading 25-06-2024_172041.png…]()

**Comment by DreamlyJack** [comment]
it remains to be done also with a striped dress, and with a turtleneck dress, the yellow old dress will remain the same tone, because other colors do not suit her

**Pull request closed without merging**